packages/vc-verification/src/role-resolvers.ts

```diff
@@ -0,0 +1,66 @@
+import {
+  IIssuerDefinition,
+  IRoleDefinitionV2,
+} from '@energyweb/credential-governance';
+import { providers } from 'ethers';
+import { RoleDefinitionResolverV2__factory } from '@energyweb/credential-governance/ethers/factories/RoleDefinitionResolverV2__factory';
+import { RoleDefinitionResolverV2 } from '@energyweb/credential-governance/ethers/RoleDefinitionResolverV2';
+
+abstract class RoleResolver {
+  abstract getRoleIssuers(namespace: string): Promise<IIssuerDefinition>;
+}
```

Comment:
In general, I think it is better to use interfaces rather than abstract classes as a class can implement multiple interfaces.

Maybe call it `IssuerDefinitionResolver` as it looks like it is IssuerDefinitions that are being resolved?

```suggestion
interface class IssuerDefinitionResolver {
  getRoleIssuerDefinition(namespace: string): Promise<IIssuerDefinition>;
}
```